force_torque_sensor_controller/include/force_torque_sensor_controller/force_torque_sensor_controller.hpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2021, Stogl Robotics Consulting UG (haftungsbeschränkt)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef FORCE_TORQUE_SENSOR_CONTROLLER__FORCE_TORQUE_SENSOR_CONTROLLER_HPP_
+#define FORCE_TORQUE_SENSOR_CONTROLLER__FORCE_TORQUE_SENSOR_CONTROLLER_HPP_
+
+#include <string>
+#include <vector>
+
+#include "controller_interface/controller_interface.hpp"
+#include "force_torque_sensor_controller/visibility_control.h"
+#include "geometry_msgs/msg/wrench_stamped.hpp"
```

Comment:
please make sure to declare this as a dependency in the cmake and in the package.xml as well

Reply:
Dependency has been added. Please check once and provide feedback.